_data/domains.yml

```diff
@@ -206,6 +215,7 @@ websites:
       img: heartinternet.png
       tfa: No
       twitter: HeartInternet
+      facebook: heartinternet
```

Comment:
Looks good, but do you think they know that the Facebook icon on their site actually directs to their LinkedIn page?